File: minions/10.Sucesión la Bicicleta/Sucesion_la_Bicicleta.py
def main():
    b0, b1, bn = 2.0, 3.0, 0.0
    i, n= 0, 0
    
    while n < 3:
        n = int(input("Ingrese un numero (debe ser >= 3) : "))
        if n >= 3:
            break
        else:
            print("Error, debe ser >= 3 ")
            
    b0,b1= 2.0, 3.0
    print("{}\n{}".format(b0, b1))
    
    for i in range(1, n - 1):
        bn = (b1 + 1)/b0
        print("{}".format(bn))
        b0, b1 = b1, bn
    

if __name__ == "__main__":
    main()
# devuelve con decimales y en los casos prueba no aparecen con decimales

File: Ejercicios/10.Sucesión la Bicicleta/Sucesion_la_Bicicleta.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    float b0 = 2, b1 = 3, bn;
    int i, n;

    do{
        scanf("%d", &n);
        if (n < 3){
            printf("Error, debe ser >=3\n");
        }   
    } while (n < 3);

    printf("%3.1f\n%3.1f\n", b0, b1);

    for (i = 1; i <= n - 2; i += 1){
        bn = (b1 + 1) / b0;
        printf("%3.1f\n", bn);

        b0 = b1;
        b1 = bn;
    }

    return EXIT_SUCCESS;
}